src/components/panel/panel.js

```diff
@@ -757,8 +807,8 @@ MdPanelService.prototype.create = function(config) {
     return this._trackedPanels[config.id];
   }
 
-  // If no ID is set within the passed-in config, then create an arbitrary ID.
   this._config = {
+    // If no ID is set within the passed-in config, then create an arbitrary ID.
     id: config.id || 'panel_' + this._$injector.get('$mdUtil').nextUid(),
```

Comment:
I think `$mdUtil` should be injected directly like `$window` and `$rootScope` in the constructor. `$injector` was only meant to be used when creating a new panelPosition or panelAnimation, which also require injected variables.